proximal/examples/test_metric.py

```diff
@@ -83,14 +83,14 @@
 plt.show()
 
 #Error
-print('Maximum error NLM (CUDA vs. CPU) {0}'.format( np.amax( np.abs( output - dst ) ) ) ) 
+print('Maximum error NLM (CUDA vs. CPU) {0}'.format( np.amax( np.abs( output - dst ) ) ) )
```

Comment:
These changes are due to my editor (spyder) automatically formatting to remove extra spaces at the end of lines. Could be removed from the commit if you want.